### DISCUSSION TITLE
Creepers cause Lingering Potion Effects when exploding

### DISCUSSION THREAD

**Discussion by GregoriusT** [Roadhog360/Et-Futurum-Requiem#180] [Ideas]
So recently I found out that if a Creeper explodes, and said Creeper has Potion Effects on them for whatever reason (like if you use a Splash Potion), it will leave a small puddle of Lingering Potion around the center of the Explosion.

And apparently this Feature exists since Minecraft 1.11, according to the [Wiki](https://minecraft.fandom.com/wiki/Creeper)

I recently saw a Video where that Feature was made use of, and when I tried it out with a recent™ Version of Et Futurum Requiem, nothing happened.

**Comment by Roadhog360** [comment]
Fulfilled in [be026d2](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/be026d2b9480ac6e6f41979b338a638e21348f45).
